bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/ImageImpl.java

```diff
@@ -110,13 +111,15 @@ protected void initModel() {
             disableLazyLoading = currentStyle.get(PN_DESIGN_LAZY_LOADING_ENABLED, true);
 
             String staticSelectors = selector;
+            final String imageName = getImageNameFromDam();
```

Comment:
How does this work now for locally uploaded assets now? From where to extract the name?

Reply:
> @kwin, we won't retrieve a name for local uploads

@kwin  @richardhand  Just to confirm it, in that case the image URL would be with the timestamp as it is in current version.
`<image-component-path>.<selectors>.<extension>/<timestamp>.<extension>`

Reply:
@ompandeyy yes exactly